### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by nicioniz** [theawless/EventLock#14]
Hi!
Thanks a lot for this helpful module.
It would be great if we can have a translation for the words displayed in the lockscreen (today, tomorrow...).
It also would be great if we can decide the "at" string (the string that concat title and place). I'd like a lot to use the @ 😄 
Thanks for your work!

**Comment by theawless** [comment]
I don't have resources to translate the application. But I can definitely let the user edit the "at", "today", etc, strings.

**Comment by Latschenharry** [comment]
I would assist you with the german translation. Just contact me :)

**Comment by theawless** [comment]
That's great! Have you translated an android app before?

https://github.com/theawless/EventLock/blob/master/app/src/main/res/values/strings.xml
It's pretty easy. You just have to translate the strings in this file. Do make a pull request :smile:

**Comment by Latschenharry** [comment]
I made an app myself already so this should go well. :)

**Comment by nicioniz** [comment]
Hi. I think for the "at" it  would be great to let the user change it.
For the other strings I can do the Italian translation, but I don't know how to use github

**Comment by theawless** [comment]
@nicioniz Yes, I will give options for that. As for the translation, it would be really helpful if you translate the string in this file.
https://github.com/theawless/EventLock/blob/master/app/src/main/res/values/strings.xml

Then you can add it here as an attachment.

**Comment by Ekky78** [comment]
Hello. also I like to help with translation into Czech. and I would ask the color of the text on the locked screen. text appears white and I would welcome the option to black. thanks for the reply.

**Comment by nicioniz** [comment]
Hi! Sorry for all the time passed, but I was on exams.

Here there's the Italian translation. I tried to adjust all the phrases to the context where they are used. I hope it'll be good.
I attach the file in txt type because I can't attach an xml file.
[strings.xml.txt](https://github.com/theawless/EventLock/files/764534/strings.xml.txt)

**Comment by Dil3mm4** [comment]
If the development it's still running, I can offer Italian translation.

**Comment by theawless** [comment]
@nicioniz  thanks for your support! Sorry for the delay, I was extremely busy with university.

Commit e73e0c3e1e8e91dcd4e21dd93e7560f4a6830371 allows for translating the displayed words like "at", "All day", etc.

**Comment by theawless** [comment]
The development is running again. I am currently working on the latest update. Any help will be appreciated and merged in this update.

Please attach the translated strings.xml here or create a pull request if possible.
